test/e2e_node/memory_manager_test.go

```diff
@@ -309,6 +352,32 @@ var _ = SIGDescribe("Memory Manager [Serial] [Feature:MemoryManager]", func() {
 		framework.ExpectEqual(numaNodeIDs, currentNUMANodeIDs.ToSlice())
 	}
 
+	waitingForHugepages := func(hugepagesCount int) {
```

Comment:
what's the benefit of defining the function inline and not as named function (with more args ofc)?